conf/pf.conf.defaults

```diff
@@ -1290,6 +1290,11 @@ acct_maintenance_interval=60s
 #
 # Interval at which PacketFence runs the polling enforcement on the provisioners
 provisioning_compliance_poll_interval=300s
+#
+# maintenance.online_node_state_update_interval
```

Comment:
online_node_count_interval would be more appropriate I think.

State could mean registered vs unregistered or some other things.

Especially some other things...